### PR TITLE
chores: prettier config

### PR DESCRIPTION
**Pull request by CGQAQ**
* add prettier to recommand extensions
* prettier config is now required
* single quotes to double quotes
* more specifically what files will be formatted by prettier
* add prettier check for later CI uses

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "singleQuote": true,
+  "singleQuote": false,
```
Comment (justjavac) [inline .prettierrc:2]: i5ting loves single

**Review by justjavac — CHANGES_REQUESTED** [review]

File: .prettierrc (hunk 1 above)
Comment (liruifengv) [reply .prettierrc:2]: +1

Comment (CGQAQ) [reply .prettierrc:2]: no, he has already been married

Comment (justjavac) [reply .prettierrc:2]: https://github.com/i5ting/ts-junit/commit/3fabaa8a0a5bdae4dc9f9dda3798cd95bd7b88ed#diff-a5de3e5871ffcc383a2294845bd3df25d3eeff6c29ad46e3a396577c413bf357R27

**Pull request merged**